ext/civicrm_admin_ui/ang/afsearchEvents.aff.html

```diff
@@ -0,0 +1,9 @@
+<div af-fieldset="">
+  <div class="af-markup">
+    <div class="help">
+    <p>This page lists all event registrations for this contact since inception. Click Add Event Registration below to register this contact for an event. Click Submit Credit Card Event Registration to process a new registration on behalf of the participant using their credit card.</p>
+    </div>
```

Comment:
We need to add `ts()` around this test. I started doing that, and that got me thinking about the text itself and how it's a big mouthful of nothing. It just explains a self-explanatory table, then tells you you can click on some self-explanatory buttons and that they will do the thing they already say they will do.